package.json

```diff
@@ -1,7 +1,7 @@
 {
   "productName": "SikhiToTheMax",
   "name": "sttm-desktop",
-  "version": "5.3.2",
+  "version": "6.0.0",
```

Comment:
should we do 5.3.3 for that overlay bug?

Reply:
Agreed.